types/nginstack__datasource/index.d.ts

```diff
@@ -0,0 +1,29 @@
+// Type definitions for @nginstack/datasource 50.0
+// Project: https://dev.azure.com/nginstack/nginstack
+// Definitions by: Renato Ribeiro <https://github.com/Henato>
+//                 Paulo Moreno <https://github.com/paulocmoreno>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+// Minimum TypeScript Version: 4.3
```

Comment:
what 4.3 features do these definitions use? Can you remove them to support earlier TS versions?

Reply:
It is expected that our user base is using 4.3, but it's not strictly necessary. We still need recursive type aliases from 3.7 though. We could set it to 3.7 minimum, no problem.